src/configurationmanager.cpp

```diff
@@ -624,11 +631,29 @@ void ConfigurationManager::loadConfiguration(bool debug)
             fi.close();
         }
 
-        //Save version
+
+        //Check if version has been updated.
         stringstream versionionfile;
-        versionionfile << configFolder << "/" << "megacmd.version";
-        ofstream fo(versionionfile.str().c_str(), ios::out);
+        versionionfile << configFolder << "/" << VERSION_FILE_NAME;
+
+        // Get latest version if any.
+        string latestVersion;
+        ifstream fiVer(versionionfile.str().c_str());
+        if (fiVer.is_open())
+        {
+            fiVer >> latestVersion;
+            fiVer.close();
+        }
+
+        if (latestVersion.size()>0 && (MEGACMD_CODE_VERSION > stol(latestVersion)))
+        {
+            hasBeenUpdated = true;
+            if (debug)
+                cout  << "MEGAcmd has been updated." << endl;
+        }
```

Comment:
Picky comments to follow coding style (although surrounding one wasn't following it ;-) ):
preferable: `latestVersion.size()>0` -> `!latestVersion.empty()`, or `latestVersion.size()`   or `latestVersion.size() > 0` in any case.
`cout` -> `std::cout`, same for endl
and surround the if body-code between '{' and '}